index.bs

```diff
@@ -6595,9 +6595,6 @@ There are <a>channelCount constraints</a> and <a>channelCountMode
 constraints</a> for this node. These constraints ensure that the
 input to the node is either mono or stereo.
 
-{{ConvolverNode}}s are created with an internal flag <code>buffer
```

Comment:
This is basically repeated in the first sentence in the [constructor](https://webaudio.github.io/web-audio-api/#dom-convolvernode-convolvernode), so I'm removing this redundant sentence